include/psp2/sysmodule.h

```diff
@@ -90,9 +90,17 @@ enum {
 	SCE_SYSMODULE_AVPLAYER              = 0x004c
 };
 
+/* internal module IDs */
+enum {
+	SCE_SYSMODULE_PROMOTER_UTIL         = 0x80000024
+};
+
 int sceSysmoduleLoadModule(SceUInt16 id);
 int sceSysmoduleUnloadModule(SceUInt16 id);
 int sceSysmoduleIsLoaded(SceUInt16 id);
+int sceSysmoduleLoadModuleInternal(SceUInt32 id);
```

Comment:
Here it the format of the comments : https://github.com/vitasdk/vita-headers/blob/540158d9b47ef8e875bee65d3a8baee4fffe7cbc/include/psp2/ctrl.h#L148-L156

I'll push a commit.